sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -127,7 +127,7 @@ case class CatalogTable(
     sortColumnNames: Seq[String] = Seq.empty,
     bucketColumnNames: Seq[String] = Seq.empty,
     numBuckets: Int = -1,
-    owner: String = "",
+    owner: String = System.getProperty("user.name"),
```

Comment:
User name will be complicated. It is set from current SessionState authenticator. I do not believe we have reached that yet. I will revert this part.